extensions/BugModal/web/bug_modal.js

```diff
@@ -1389,6 +1389,44 @@ $(function() {
             saveBugComment(event.target.value);
         });
 
+    // Allow to attach pasted text directly
+    document.querySelector('#comment').addEventListener('paste', event => {
+      const text = event.clipboardData.getData('text');
+      const lines = text.split(/(?:\r\n|\r|\n)/).length;
+
+      if (lines < 50) {
+        return;
+      }
+
+      const extract = text.replace(/(?:\r\n|\r|\n)/, ' ').trim().substr(0, 20);
+      const summary = (window.prompt(`You’re pasting ${lines} lines of text starting with “${extract}”. Enter the ` +
+        'summary below and click OK to upload it as an attachment. Click Cancel to paste it normally.') || '').trim();
```

Comment:
Nit: insert blank space above "Enter the summary below..." It is less confusing when it is more obvious what it is expected of the user to do with the empty text field.

Reply:
Fixed!